tbump/test/test_file_bumper.py

```diff
@@ -21,6 +21,10 @@ def test_file_bumper_simple(test_repo: Path) -> None:
     assert file_contains(test_repo / "glob-one.c", 'version_one = "1.2.41-alpha-2"')
     assert file_contains(test_repo / "glob-two.v", 'version_two = "1.2.41-alpha-2"')
 
+    # one occurence is bumped up, others remain unchanged
+    assert file_contains(test_repo / "Cargo.toml", 'version = "1.2.41-alpha-2"')
```

Comment:
To simplify my life, I did not created a separate test for checking multi-line replacement results. So I've re-used this one here to check that at least one line has been bumped, and at least one did not.
This is not 100% accurate assertion.
It rather should be:
- exactly first occurrence of version string has been changed, and
- two other occurrences did not

Again, saving time here. Feel free to extend this check to more strict if you like.

Reply:
I think we really need a separate test. The goal of a test suite is to have exactly one test failing if there's a bug somewhere, so it would be better to have a dedicated test for the multiline case.

And of course, the test should check the entire contents of the replaced file :)